x/gov/migrations/v5/store_test.go

```diff
@@ -43,7 +43,8 @@ func TestMigrateStore(t *testing.T) {
 	bz = store.Get(v4.ParamsKey)
 	require.NoError(t, cdc.Unmarshal(bz, &params))
 	require.NotNil(t, params)
-	require.Equal(t, v1.DefaultParams().ExpeditedMinDeposit, params.ExpeditedMinDeposit)
+	// After migration, expect `ExpeditedMinDeposit` to equal previous value of `MinDeposit`.
+	require.Equal(t, params.MinDeposit, params.ExpeditedMinDeposit)
 	require.Equal(t, v1.DefaultParams().ExpeditedThreshold, params.ExpeditedThreshold)
 	require.Equal(t, v1.DefaultParams().ExpeditedVotingPeriod, params.ExpeditedVotingPeriod)
```

Comment:
shouldn't some of these values change?

Reply:
These lines weren't updated since both `v1.DefaultParams().XXXX` and `params.ExpeditedVotingPeriod` were updated. I changed them to literals to be explicit